web/src/features/fba/components/FuelTypesBreakdown.tsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react'
+import { Grid, Paper, Typography } from '@mui/material'
+import makeStyles from '@mui/styles/makeStyles'
+import { isUndefined } from 'lodash'
+import { FireZone, FireZoneThresholdFuelTypeArea } from 'api/fbaAPI'
+import { PieChart, Pie, ResponsiveContainer, Cell } from 'recharts'
+
+const useStyles = makeStyles({
+  wrapper: {
+    minWidth: 400
+  },
+  fuelTypesPaper: {
+    padding: '20px 10px'
+  },
+  fuelTypesHeader: {
+    fontSize: '1.3rem',
+    textAlign: 'center',
+    variant: 'h3'
+  },
+  pieChartHeader: {
+    fontSize: '1rem',
+    variant: 'h4'
+  }
+})
+
+interface Props {
+  className?: string
+  selectedFireZone: FireZone | undefined
+  fuelTypeInfo: Record<number, FireZoneThresholdFuelTypeArea[]>
+}
+
+interface FuelTypeDataForPieChart {
+  area: number
+  fuel_type_code: string
+}
+
+const RADIAN = Math.PI / 180
+const COLOURS = [
+  '#2191FB',
+  '#FCB1A6',
+  '#B33951',
+  '#CCF5AC',
+  '#8CDEDC',
+  '#9DACFF',
+  '#4F7CAC',
+  '#FFA62B',
+  '#C09BD8',
+  '#EBC3DB',
+  '#D19C1D',
+  '#FFC0BE',
+  '#ED7D3A'
+]
+
+const FuelTypesBreakdown = (props: Props) => {
+  const classes = useStyles()
+
+  const renderCustomizedLabel = ({
+    cx,
+    cy,
+    midAngle,
+    innerRadius,
+    outerRadius,
+    percent,
+    fuel_type_code,
+    area,
+    index
+  }: {
+    cx: number
+    cy: number
+    midAngle: number
+    innerRadius: number
+    outerRadius: number
+    percent: number
+    fuel_type_code: string
+    area: number
+    index: number
+  }) => {
+    const radius = innerRadius + (outerRadius - innerRadius) * 0.5
+    const x = cx + radius * Math.cos(-midAngle * RADIAN)
+    const y = cy + radius * Math.sin(-midAngle * RADIAN)
+
+    if (percent * 100 < 2) {
+      return
+    }
+    if (percent * 100 < 5) {
+      return (
+        <text x={x} y={y} fontSize={'10pt'} fill="black">
+          {`${fuel_type_code} (${(percent * 100).toFixed(0)}%)`}
+        </text>
+      )
+    }
+
+    return (
+      <text x={x} y={y} fontSize={'10pt'} fill="black" textAnchor={x > cx ? 'start' : 'end'}>
+        {`${fuel_type_code}: ${area.toLocaleString(undefined, { maximumFractionDigits: 0 })} ha (${(
```

Comment:
I think Neal mentioned the area portion of the label can be removed.

Reply:
I didn't incorporate this change because I'm a rogue 😈 And also because I didn't want to have to wait to go through the whole dev deployment process yet again. @conbrad when your merge main into your branch once this PR is merged in, feel free to delete the area part of the label if y'all think we should. (Personally, I think there's some value in displaying that area value...)